server/api/db/sqldb/db.py

```diff
@@ -1753,10 +1754,14 @@ def store_function(
             function.setdefault("metadata", {})["name"] = name
         fn = self._get_class_instance_by_uid(session, Function, name, project, uid)
         if not fn:
+            foo_spec = function.get("spec", {}).get("foo")
             fn = Function(
                 name=name,
                 project=project,
                 uid=uid,
+                foo_spec=FunctionSpecFoo(
+                    data=foo_spec,
+                ),
```

Comment:
what if `foo_spec` is `None`? should we still add it here?

Reply:
Also, if the function - what will happen to the corresponding `FunctionFooSpec` record?